workflows/prognostic_c48_run/docs/local-execution.rst

```diff
@@ -77,6 +77,27 @@ the end of the previous one will be appended. For example, the following for loo
 
         runfv3 append gs://path/to/failing/run
 
+.. note::
+
+    An option "mpi_launcher" has been added to the subcommand "append". This option 
+    allows users to configure the segmented runs on High Perfromance Computing
+    systems and cloud platfroms. The supported values are "mpirun" and "srun".
```

Comment:
I recommend putting `mpi_launcher` and `append` in backticks, so they will be formatted as code in the documentation (see the section on ["Inline markup" in the Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup)).  I edited the text a bit too to be more specific about what this option does.
```suggestion
    An option ``mpi_launcher`` has been added to the subcommand ``append``. This
    option allows users to specify the system-specific MPI launcher used when running
    the model. By default it takes the value "mpirun", which is relevant for running
    in Docker. The only other currently supported value is "srun", which is relevant
    when running on SLURM-based High Performance Computing (HPC) systems. 
```